README.md

```diff
@@ -46,12 +46,128 @@ The `main` and `db` packages contain integration tests that require a successful
 ### No-scan domains
 In case of complaints or abuse, we may not want to continually scan some domains. You can set the environment variable `DOMAIN_BLACKLIST` to point to a file with a list of newline-separated domains. Attempting to scan those domains from the public-facing website will result in error codes.
 
-## API
+## Scan API
 
+Our API objects can look a bit complicated! There's lots of information contained in a TLS scan.
 To request a scan:
 ```
 POST /api/scan
+  { "domain": "example.com" }
+```
+
+SLet's break down exactly what each part of this giant nested response means. All API responses, not just scans, are wrapped in a JSON, like:
+```
+{
+    status_code: 200,
+    message: "",
+    response: <reponse data>
+}
+```
+Or even:
+```
+{
+    status_code: 400,
+    message: "query parameter domain not specified",
+    response: {}
+}
+```
+The status codes correspond with regular HTTP errors, and `message` provides more context into particular failures.
```

Comment:
Do they correspond with the HTTP status that the underlying service that was scanned returned, or do they summarize failures separately (like improper use of the API, in this example), or sometimes one and sometimes the other?

Reply:
If a scan isn't performed (or a new scan isn't performed) due to these limits, will the API just return older/empty scan result information, as opposed to specifically saying "you're not allowed to scan this host now" or something?